server/src/main/java/org/opensearch/action/admin/cluster/snapshots/restore/RestoreSnapshotRequest.java

```diff
@@ -164,6 +166,12 @@ public RestoreSnapshotRequest(StreamInput in) throws IOException {
         if (in.getVersion().onOrAfter(Version.V_2_17_0)) {
             sourceRemoteTranslogRepository = in.readOptionalString();
         }
+        if (in.getVersion().onOrAfter(Version.V_3_0_0)) {
+            renameAliasPattern = in.readOptionalString();
+        }
+        if (in.getVersion().onOrAfter(Version.V_3_0_0)) {
+            renameAliasReplacement = in.readOptionalString();
+        }
```

Comment:
These version checks imply it will not be in 2.18.0, but you added the changelog in 2.x section implying they will.  Can you clarify?

Reply:
I assume it can/should be in 2.18, but using V_2_18_0 caused the :qa:mixed-cluster:v2.18.0#mixedClusterTest to fail. You can see this failure in [one of the earlier Gradle checks](https://build.ci.opensearch.org/job/gradle-check/49318/consoleFull). I assume because this change is not back-ported into 2.18 yet. Back porting it I would assume would require this PR to be merged first. But merging this PR I assume would require that failing tests to pass. I am not sure how to resolve this catch-22.

Reply:
Looks like previously it's handled by putting Version.CURRENT in the initial `main` PR, then once it's backported, updating the version in the backport and forward-porting:
https://github.com/opensearch-project/OpenSearch/commit/7c9c01d8831f57b853647bebebd8d91802186778

Reply:
Ok, it has been updated with `Version.CURRENT`.